csfieldguide/chapters/content/en/big-data/sections/collecting-big-data.md

```diff
@@ -0,0 +1,130 @@
+
+{image file-path="img/chapters/google-data-centre.png" caption="true" alt="Image shows a long row of server racks at a Google data centre in Oklahoma"}
+
+Server racks in the Google data centre in Oklahoma.
+
+{image end}
+
+These data centres store a mind-boggling amount of information. Just think, every video uploaded to YouTube, every tweet ever written, every purchase ever made on Amazon, and every comment on every blog on the internet exists somewhere in one of these data centres (in fact it will exist in several, as there will be multiple backups of it).
+
+{panel type="curiosity"}
+
+# A Data Centre on Punch Cards
+
+In 2013, xkcd creator Randall Munroe tried to answer the question [“If all digital data were stored on punch cards, how big would Google's data warehouse be?”](https://what-if.xkcd.com/63/) and with some clever internet-sleuthing calculated that over a quadrillion cards would be needed. It would be enough to cover all six states of New England in 4.5km of punch cards. And that was all the way back in 2013! Imagine how much bigger that pile would be now!
+
+{image file-path="img/chapters/punchcard-layer-google.png" caption="true" alt="Image shows an illustration of how high the stack of punch cards holding Google's data would be in relation to the city of Boston. The layer of punch cards is more than 10 times higher than the buildings in Boston."}
+
+Image courtesy of [xkcd](https://xkcd.com/).
+
+{image end}
+
+{panel end}
+
+The other issue around storing Big Data is the raw data itself.
+There are two general categories we use to classify data: structured or unstructured.
+When you describe the qualities of a car, you probably list the colour, make, model, whether it is manual or automatic, etc.
+We would call this data structured because there is a predetermined data model we can use to describe the qualities of the car.
+For unstructured data however, there is no predetermined data model.
+This is why it cannot easily be poured directly into a typical database, and therefore needs data crunching/preprocessing before it can be stored and used, which takes computing power in itself!
+Some common examples of unstructured data are images in the form of a list of pixels, or recordings of human speech used for natural language processing.
+Data is often preprocessed before being stored but storing the raw data, not just processed data, is very important for some applications.
+
+{panel type="jargon-buster"}
+
+# Data Crunching
+
+Data Crunching, also called preprocessing, means the process by which a large amount of raw data is prepared for automated processing (see the following Analysis and Visualisation sections).
```

Comment:
This implies crunching is not automated, is this correct?

Add links to the two sections and perhaps say "This is explained further in" instead of "see the"

Reply:
Good point, I'll have a look into how to describe data crunching in this context and post a new version